app/src/common/chromium/com/igalia/wolvic/browser/api/impl/SettingsImpl.java

```diff
@@ -73,7 +76,7 @@ public boolean getFullAccessibilityTree() {
 
     @Override
     public void setUserAgentMode(int value) {
-        // TODO: implement
+        mSessionSettings.setUserAgentMode(value);
```

Comment:
As suggested in code review, we should not depend in chromium backend side on the specific values of the user agent mode defined in wolvic side.

Here we should convert the wolvic values to the chromium backend values.